### DISCUSSION TITLE
CRBN Gear

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1388] [Question and answer] [answered]
_Hello!_
I would like to know if it is possible to add helmets as CRBN gear. Thanks in advance for the answer!

**Comment by Vdauphin** [comment, marked as answer]
Hello,

Not directly because we are not checking the helmet type
You can try here https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/chem/damage.sqf

Cheers
